server/start.go

```diff
@@ -248,7 +248,7 @@ func startStandAlone(ctx *Context, appCreator types.AppCreator) error {
 	}
 
 	defer func() {
-		if err = svr.Stop(); err != nil {
+		if err = errors.Join(svr.Stop(), app.Close()); err != nil {
```

Comment:
we have to use the hashicorp errors package here i believe because this version go doesnt support .Join